ksql-parser/src/main/antlr4/io/confluent/ksql/parser/SqlBase.g4

```diff
@@ -77,9 +77,15 @@ query
       (WHERE where=booleanExpression)?
       (GROUP BY groupBy)?
       (HAVING having=booleanExpression)?
+      ((EMIT | WITH) resultMaterialization)?
```

Comment:
🤔 using `WITH` here seems weird because I'll have `CREATE STREAM foo WITH (PROPS) AS SELECT foo WITH CHANGES`?

Reply:
i refer the right honourable gentleman to the first comment in the description:

> NOTE: this may not be the final syntax we go with, but it's good enough for now.

Reply:
Update: it's now been changed to `YIELD`

However, let's keep any discussion on the _right_ keyword out of the PR and in the KLIP.

Reply:
yup yup! I promise I did read the description, I just wanted to write that down while i remembered it. I do like YIELD more